### DISCUSSION TITLE
Tickets module - Status bar

### DISCUSSION THREAD

**Discussion by afatac** [YetiForceCompany/YetiForceCRM#16943] [How to] [answered]
At the demo site, I see this status bar in Tickets module. But I cannot see it in our Tickets module. How to get it?

![image](https://user-images.githubusercontent.com/26587885/223376710-49089b86-d595-4356-bfd2-1c90a00ed403.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://gitdeveloper.yetiforce.com/index.php?module=LayoutEditor&parent=Settings&view=Index&sourceModule=HelpDesk
![image](https://user-images.githubusercontent.com/10198654/223377215-9bb5a440-b948-4a57-b3e4-f455daeb80c1.png)
